CHANGELOG.md

```diff
@@ -14,16 +39,114 @@
   +<Select value="Outlined" />
   ```
 
-- [FormControl] Change default variant from standard to outlined (#24895) @petyosi
-  Standard has been removed from the Material Design guidelines. [This codemod](https://github.com/mui-org/material-ui/tree/next/packages/material-ui-codemod#variant-prop) will automatically update your code.
+#### Changes
+
+- <!-- 17 --> [Alert] Fix action to be aligned with the text (#25768) @mnajdova
+- <!-- 30 --> [Avatar] Fix onload event not firing when img cached (#25793) @npandrei
+- <!-- 35 --> [Box] Add utility mui class (#25802) @mnajdova
+- <!-- 24 --> [core] Don't call noop event.persist() (#25782) @eps1lon
```

Comment:
Should it be moved to the `### Core` section? It seems to be a pit polish

Reply:
Oh ok, I see https://github.com/mui-org/material-ui/pull/25849#discussion_r616504509